metadata.rb

```diff
@@ -3,15 +3,11 @@
 maintainer_email  'help@sous-chefs.org'
 license           'Apache-2.0'
 description       'Installs and maintains php and php modules'
-source_url        'https://github.com/sous-chefs/php'
-issues_url        'https://github.com/sous-chefs/php/issues'
+source_url        'https://github.com/ArtofBugs/php'
+issues_url        'https://github.com/ArtofBugs/php/issues'
```

Comment:
Should I skip the metadata test then? Since the integration suites are dependent on the lint-unit, they won't run at all if any part of the lint-unit fails.

Reply:
You know, it might be a good idea to just make a proper draft PR in the Sous Chefs php repo. That way there's visibility. Would you be open to doing that?

Reply:
Sure! I'll make the changes you suggested and then open a PR.

Reply:
Opened a PR yesterday! https://github.com/sous-chefs/php/pull/350